new-docs/components/Card.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
```

Comment:
@Ray2126 I know this is merged but some feedback would be that this should be an anchor tag 👍

Reply:
Would this avoid needing an onClick method?

Reply:
@AndyClifford yes

Reply:
Thanks @MeganSteenkamp , I have a PR up to replace the div with a `NuxtLink` which is Nuxt's version of an anchor tag.

Will change the `h6` and `header` tags as well :)